WhaleSpotting/Services/UserService.cs

```diff
@@ -28,4 +29,8 @@ public User GetById(int id)
     {
         return _users.GetById(id);
     }
+    public User CreateNewUser(CreateUserRequest newUser)
+    {
+        return _users.CreateNewUser(newUser);
+    }
```

Comment:
Same here, we can use the existing `Create` function

Reply:
Fixed